airflow-core/src/airflow/ui/public/i18n/locales/es/common.json

```diff
@@ -0,0 +1,280 @@
+    "dataIntervalStart": "Intervalo de Datos Inicial",
+    "lastSchedulingDecision": "Última Decisión de Programación",
+    "queuedAt": "En Cola en",
+    "runAfter": "Ejecutar Después",
+    "runType": "Tipo de Ejecución",
+    "sourceAssetEvent": "Evento de Asset Fuente",
+    "triggeredBy": "Activado por"
+  },
+  "dagRun_one": "Ejecución del DAG",
+  "dagRun_other": "Ejecuciones del DAG",
+  "dagWarnings": "Advertencias/Errores del DAG",
+  "defaultToGraphView": "Por defecto a vista gráfica",
+  "defaultToGridView": "Por defecto a vista en cuadrícula",
+  "direction": "Dirección",
+  "docs": {
+    "documentation": "Documentación",
+    "githubRepo": "Repositorio de GitHub",
+    "restApiReference": "Referencia de REST API"
+  },
+  "duration": "Duración",
+  "endDate": "Fecha de Fin",
+  "error": {
+    "back": "Atrás",
+    "defaultMessage": "Ocurrió un error inesperado",
+    "home": "Inicio",
+    "notFound": "Página no encontrada",
+    "title": "Error"
+  },
+  "expression": {
+    "all": "Todos",
+    "and": "Y",
+    "any": "Cualquiera",
+    "or": "O"
+  },
+  "logicalDate": "Fecha Lógica",
+  "logout": "Cerrar Sesión",
+  "logoutConfirmation": "Estás a punto de cerrar sesión de la aplicación.",
+  "mapIndex": "Mapa de Índice",
+  "modal": {
+    "cancel": "Cancelar",
+    "confirm": "Confirmar",
+    "delete": {
+      "button": "Eliminar",
+      "confirmation": "¿Estás seguro de querer eliminar {{resourceName}}? Esta acción no se puede deshacer."
+    }
+  },
+  "nav": {
+    "admin": "Administración",
+    "assets": "Assets",
+    "browse": "Navegar",
+    "dags": "DAGs",
+    "docs": "Docs",
+    "home": "Inicio",
+    "plugins": "Plugins",
+    "security": "Seguridad"
+  },
+  "noItemsFound": "No se encontraron {{modelName}}s",
+  "note": {
+    "add": "Agregar una nota",
+    "dagRun": "Nota de Ejecución del DAG",
+    "label": "Nota",
+    "placeholder": "Agregar una nota...",
+    "taskInstance": "Nota de Instancia de Tarea"
+  },
+  "pools": {
+    "deferred": "Diferido",
+    "open": "Abierto",
+    "pools_one": "pool",
+    "pools_other": "pools",
+    "queued": "En Cola",
+    "running": "En Ejecución",
+    "scheduled": "Programado"
+  },
+  "runId": "ID de la corrida",
+  "runTypes": {
+    "asset_triggered": "Asset Activado",
+    "backfill": "Backfill",
+    "manual": "Manual",
+    "scheduled": "Programado"
+  },
+  "scroll": {
+    "direction": {
+      "bottom": "abajo",
+      "top": "arriba"
+    },
+    "tooltip": "Presiona {{hotkey}} para desplazarte a {{direction}}"
+  },
+  "seconds": "{{count}}s",
+  "security": {
+    "actions": "Acciones",
+    "permissions": "Permisos",
+    "resources": "Recursos",
+    "roles": "Roles",
+    "users": "Usuarios"
+  },
+  "selectLanguage": "Seleccionar Idioma",
+  "showDetailsPanel": "Mostrar Panel de Detalles",
+  "sourceAssetEvent_one": "Evento de Asset Fuente",
+  "sourceAssetEvent_other": "Eventos de Asset Fuente",
+  "startDate": "Fecha de Inicio",
```

Comment:
@m1racoli Maybe we change all Start Dates to "Fecha Inicial" and End Date to "Fecha Final"